mozfun/glean/normalize_baseline_client_info/metadata.yaml

```diff
@@ -0,0 +1,3 @@
+description: Accepts a glean client_info struct as input and returns a modified struct
```

Comment:
If we implement a `norm` dataset as suggested below, I'd argue that this should be moved there as `norm.glean_baseline_client_info`.